src/index.html

```diff
@@ -11,8 +11,51 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__photo"></div>
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__code">Product code: 195434</p>
+      <div class="stars">
+        <div class="stars__total stars__total--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="stars__number-Of-Reviews">Reviews: 5</p>
+      </div>
+      <div class="card__price-block">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <button class="card__button">
+        <a
+          data-qa="hover"
+          href="#"
+        >
+          buy
+        </a>
```

Comment:
The 'a' tag should not be nested inside the 'button' tag. It's a bad practice because it can lead to unpredictable behavior. You should use either 'a' or 'button', depending on the situation. If you need a link that looks like a button, use the 'a' tag and style it accordingly. If you need a button (for example, to submit a form), use the 'button' tag.